.ci/pitest.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 function list_profiles() {
   POM_PATH="$(dirname "${0}")/../pom.xml"
-  cat "$POM_PATH" | sed -n -e 's/^.*<id>\(pitest-[^<]\+\)<\/id>.*$/\1/p' | sort
+  cat "$POM_PATH" | sed -E -n -e 's/^.*<id>(pitest-[^<]+)<\/id>.*$/\1/p' | sort
```

Comment:
Should we remove `-e`? I know that "e" is regular expression in some tools, but I will never remember what tool and case of this parameter.

https://www.gnu.org/software/sed/manual/sed.html

>-e script
--expression=script
Add the commands in script to the set of commands to be run while processing the input.

Reply:
Can we  remove `-n` ?

>-n
--quiet
--silent
By default, sed prints out the pattern space at the end of each cycle through the script (see [How sed works](https://www.gnu.org/software/sed/manual/sed.html#Execution-Cycle)). These options disable this automatic printing, and sed only produces output when explicitly told to via the p command.

Reply:
We can't remove `-n`, here's `man sed` on Mac, removing it will pring the whole pom.xml into output followed by a list of profiles:
> -n      By default, each line of input is echoed to the standard output after all of the commands have been applied to it.  The -n option suppresses this behavior.

Removed `-e` as it is explicit in our case.

Reply:
We probably need to create separate issue to update all `sed` to have `-n` I never used such argument, so there might a lot of places which not going to work in freebsd family.

Reply:
https://github.com/checkstyle/checkstyle/issues/13078